addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -230,6 +231,15 @@ def get_livechat_info(self, username='Visitor'):
             info['options']["default_username"] = username
         return info
 
+    def _cron_clean_empty_sessions(self):
```

Comment:
Could be good to copy a little bit of your nice commit message and put it here as docstring to explain why it's needed.